malaria24/ona/models.py

```diff
@@ -207,8 +207,10 @@ def alert_new_case(sender, instance, created, **kwargs):
 
         if instance.reported_by:
             send_sms.delay(to=instance.reported_by,
-                           content=('Your reported case has been assigned '
-                                    'case number %s.' % (
+                           content=('Your reported case for %s %s has been '
+                                    'assigned case number %s.' % (
+                                        instance.first_name,
+                                        instance.last_name,
                                         instance.pk,)))
```

Comment:
Shouldn't this PR have #24 merged in and then this should become `instance.case_number` ?